README.md

```diff
@@ -113,6 +113,16 @@ python -m build --wheel # in this repository to build the wheel. Assumes you hav
 pip install dist/<generated>.whl
 ```
 
+### Vulkan support
+
+Thanks to [@thewh1teagle](https://github.com/thewh1teagle), using Vulkan is now supported:
```

Comment:
Your contribution is appreciated. Thanks.
But can you explain how `Vulkan` is now supported thanks to you  ?

You literally just updated the `whisper.cpp` submodule :sweat_smile: 
I don't see any changes to the repository itself in this regard!

Reply:
This PR updates the submodule, which includes the necessary changes to make Vulkan work. I'll remove the unnecessary part now.

Reply:
No, please keep the credits. Just remove the `now supported part`.